doc/src/modules/physics/units/philosophy.rst

```diff
@@ -275,15 +275,15 @@ Literature
 ==========
 
 .. [Page52] C. H. Page, `Classes of units in the SI
-    <https://doi.org/10.1119/1.1927482>`_,
+    <https://pubs.aip.org/aapt/ajp/article-abstract/20/1/1/1034555/Units-and-Dimensions-in-Physics?redirectedFrom=fulltext>`_,
```

Comment:
This DOI worked for me. The idea behind DOIs is that they should be the long term URL to digital documents. I think we should stick with DOIs.